contributed_definitions/yaml/NXmpes_peem.yaml

```diff
@@ -0,0 +1,114 @@
+category: application
+doc: |
+  This is a general application definition for a photo-electron emission or low energy electron microscopy/diffraction experiment.
+type: group
+(NXmpes)NXmpes_peem:
+  (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes_peem]
+    (NXmonitor):
+      exists: recommended
+    (NXinstrument):
+      (NXinsertion_device):
+          exists: optional # only when PEEM uses synchrotron light
+      (NXmonochromator): # incoming_beam_monochromator
+          exists: recommended
+      (NXbeam)beam_probe:
+        exists: optional # as we record it in NXsample
+        \@flux: # that will point to the NXmonitor data field
+        incident_energy(NX_FLOAT): # link to monochromator energy
```

Comment:
The `incident_energy` is already required in `NXmpes`, it will be inherited as required. We could keep it if you want to add to the docstring that this should be a link to the monochromator energy.

Reply:
Ok